justfile

```diff
@@ -19,11 +19,10 @@ test-unit:
 bootstrap:
     go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3
-    go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@v2.3
+    go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@latest
     go install golang.org/x/tools/cmd/godoc@latest
 gen:
-  protoc --experimental_allow_proto3_optional --proto_path=apis/proto --go_opt=module="github.com/pinecone-io/go-pinecone" --go-grpc_opt=module="github.com/pinecone-io/go-pinecone" --go_out=. --go-grpc_out=. apis/proto/pinecone/data/v1/vector_service.proto
-  oapi-codegen --package=control --generate types,client apis/openapi/control/v1/control_v1.yaml > internal/gen/control/control_plane.oas.go
+  ./codegen/build-clients.sh "2024-07"
```

Comment:
Calling the script with version from here, we could set a global "API_VERSION" constant somewhere and use that here and elsewhere maybe?

Reply:
`just` can also have variables: https://just.systems/man/en/chapter_29.html